com.unity.render-pipelines.core/Editor/CoreEditorUtils.cs

```diff
@@ -448,8 +448,8 @@ public static bool DrawSubHeaderFoldout(GUIContent title, bool state, bool isBox
         /// <param name="hasMoreOptions">Delegate saying if we have MoreOptions</param>
         /// <param name="toggleMoreOptions">Callback called when the MoreOptions is toggled</param>
         /// <returns>return the state of the foldout header</returns>
-        public static bool DrawHeaderToggle(string title, SerializedProperty group, SerializedProperty activeField, Action<Vector2> contextAction = null, Func<bool> hasMoreOptions = null, Action toggleMoreOptions = null)
-            => DrawHeaderToggle(EditorGUIUtility.TrTextContent(title), group, activeField, contextAction, hasMoreOptions, toggleMoreOptions);
+        public static bool DrawHeaderToggle(string title, SerializedProperty group, SerializedProperty activeField, string documentationURL, Action<Vector2> contextAction = null, Func<bool> hasMoreOptions = null, Action toggleMoreOptions = null)
+            => DrawHeaderToggle(EditorGUIUtility.TrTextContent(title), group, activeField, documentationURL, contextAction, hasMoreOptions, toggleMoreOptions);
```

Comment:
You are changing the signature of this method and will break everyone that is using this. 
Just load URP and it breaks.

Reply:
Oof, not sure why I wedged that right into the middle of the signature- fixed now.